docs/esi-leap-api-ref.md

```diff
@@ -10,58 +10,82 @@ The Offer API endpoint can be reached at /v1/offers.
     * This value will default to returning offers with status 'available'.
     * This value can be set to 'any' to return offers without filtering by status.
   * resource_uuid: Returns all offers with given resource_uuid.
-  * resource_type: Returns all offers with given resource_type
+  * resource_type: Returns all offers with given resource_type.
+  * resource_class: Returns all offers with given resource_class.
```

Comment:
Might be worth mentioning here (and below) that this will default to the configured "default resource class", which by default is "ironic_node".

Reply:
Mentioned.